source/openapi-admin-v3.yaml

```diff
@@ -5539,35 +5625,49 @@ components:
           enum:
             - mongodb-atlas
         config:
+          config: "null"
```

Comment:
Typo? I'm not familiar with the `config` field and not sure what "null" would mean here

Reply:
Forgot to remove that -- it was a placeholder until I figured out the details. :( Good catch!